ReduxTestTask/Classes/NewsList/ViewModel/NewsListProps.swift

```diff
@@ -14,17 +14,18 @@ extension NewsList {
         return NewsListViewController.Props(
             contentViewProps: NewsListView.ContentViewProps(
                 isLoading: state.isLoading,
-                items: state.fetchedNews.compactMap { makeNewsCellProps(from: $0) }),
+                items: state.fetchedNews.compactMap { makeNewsCellDiffableProps(from: $0) }),
```

Comment:
why compactMap?

Reply:
I forgot that compactMap for optional init